internal/auth0/connection/schema.go

```diff
@@ -853,3 +855,92 @@ var optionsSchema = &schema.Schema{
 		},
 	},
 }
+
+var scimResourceSchema = map[string]*schema.Schema{
```

Comment:
Could we move this within the `resource_scim.go`? as it's not that big and it's easier to have everything contained within the same file?

Reply:
for some reason, the schema was in a separate file, I don't really know why they did it that way originally